author: Andres Salomon <dilinger@debian.org>

Newer gn added a synonym for whitelist; but our older gn doesn't support
it.

--- a/.gn
+++ b/.gn
@@ -145,8 +145,8 @@ no_check_targets = [
 #      this situation much easier to create. if the build always lists the
 #      files and passes them to a script, it will always be correct.
 
-exec_script_allowlist =
-    build_dotfile_settings.exec_script_allowlist +
+exec_script_whitelist =
+    build_dotfile_settings.exec_script_whitelist +
     angle_dotfile_settings.exec_script_whitelist +
     [
       # Allowlist entries for //build should go into
